### PR TITLE
Update to yea 0.8 - fix yea / yeadoc issues

### PR DESCRIPTION
**Pull request by raubitsj**
Description
-----------
- Update yea to 0.8
- Fix some broken yeadoc tests

```
$ yea list
# shows the tests in the directory you are in

$ yea list --all
# shows all the tests

$ yea run
# runs the yea tests in the current directory (and subdirs)

$ yea run *.yea
# runs only the files on the command line

$ yea run --all
# runs all the tests in the repo

$ yea --yeadoc run --all
# runs all tess including yeadoc
```

These are needed first:
- https://github.com/wandb/yea/pull/29
- https://github.com/wandb/yea-wandb/pull/79

Yea 0.8.0 features:
- [x] Better probing of tests
- [x] Add strict mode to find mistakes in yea files
- [x] Fix yea permute test name issue
- [x] More consistent command line

Later:
- Add time to assert dict
- save assert dict to yea_cache
- turn on live mode
- support profiling
- rework all testids so we can make them more stable and track them overtime
- More

Testing
-------
How was this PR tested?

Checklist
-------
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -391,34 +392,36 @@ deps =
     -r{toxinidir}/requirements.txt
     func-s_{base,metaflow}-py{36,37,38,39,310}: -r{toxinidir}/requirements_dev.txt
     pytest-mock
-    yea-wandb=={env:YEA_WANDB_VERSION}
+    https://github.com/wandb/yea/archive/yea-0.8.zip
+    https://github.com/wandb/yea-wandb/archive/yea-wandb-0.8.zip
+    ; yea-wandb=={env:YEA_WANDB_VERSION}
```
Comment (kptkin) [inline tox.ini:397]: make sure to fix this!

File: tox.ini [hunk 2]
```diff
@@ -471,7 +474,9 @@ deps =
     -r{toxinidir}/requirements.txt
     standalone-{cpu,gpu,tpu,local}-py{36,37,38,39,310}: -r{toxinidir}/requirements_dev.txt
     pytest-mock
-    yea-wandb=={env:YEA_WANDB_VERSION}
+    https://github.com/wandb/yea/archive/yea-0.8.zip
+    https://github.com/wandb/yea-wandb/archive/yea-wandb-0.8.zip
+    ; yea-wandb=={env:YEA_WANDB_VERSION}
```
Comment (kptkin) [inline tox.ini:479]: same as above

**Review by kptkin — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (raubitsj) [reply tox.ini]: yea-wandb and yea were released so i have updated

File: tox.ini (hunk 2 above)
Comment (raubitsj) [reply tox.ini]: yea-wandb and yea were released so i have updated

**Pull request merged**